frontend/src/components/ModelBox/ModelConfigurationSlider.css

```diff
@@ -1,26 +1,32 @@
-.model-config-slider {
-	width: 90%;
-	margin: 0 auto;
-	padding: 0 0 0 0;
+fieldset {
```

Comment:
This will apply this styling to all over fieldsets, which is problematic if we want to add them to other components. So instead I'd rather have a class name, and prepend all other sections in this style sheet with that class name